src/components/ButtonPanel.js

```diff
@@ -1,24 +1,38 @@
 import React from 'react';
 import Button from './Button';
 
-export default class ButtonPanel extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      names: ['AC', '+/-', '%', '+',
-        '7', '8', '9', 'X',
-        '4', '5', '6', '-',
-        '1', '2', '3', '+',
-        '0', '.', '='],
-    };
-  }
+const ButtonPanel = () => (
+  <div className="panel">
+    <div>
+      <Button name="AC" />
+      <Button name="+/-" />
+      <Button name="%" />
+      <Button name="÷" color="orange" />
+    </div>
+    <div>
+      <Button name="7" />
+      <Button name="8" />
+      <Button name="9" />
+      <Button name="x" color="orange" />
+    </div>
+    <div>
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.